### PR TITLE
fix images

### PR DESCRIPTION
**Pull request by hoangvietthai**

### CODE REVIEW

**Comment by hoangvietthai** [comment]
screenshot from 2018-07-25 09-56-14

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -14,8 +14,8 @@
         <ul class="clearfix">
           <li>
             <a href="#">
```
Comment (peiguangda) [inline app/views/layouts/_header.html.erb]: link to

**Comment by peiguangda** [comment]
ok em

**Comment by NeverSayNeverLove** [comment]
good

**Pull request merged**